### ISSUE TITLE
Update 1.7.10 Forge version to latest recommended version.

### ISSUE THREAD

**Issue by ryanhaney97** [yoshiquest/forge-clj#16]
I need to get around to doing this. Minecraft Forge is currently on recommended version 10.13.4.1558, while forge-clj is on version 10.13.4.1448. I'll do this after making some tutorials.

**Comment by Immortalin** [comment]
@yoshiquest is the project still alive?

**Comment by ryanhaney97** [comment]
Oh! It was on hiatus for a while as there was a problem I couldn't figure out (that and the last weeks of college kept me preoccupied), but I fixed it a few weeks ago (and finished the spring semester of college up), so I've actually been working on it constantly. I am currently in the process of updating it to Minecraft 1.8.9. I haven't pushed anything yet as there is still a lot to do, but I currently have the blocks and items namespaces ported (as well as important namespaces like core and util), plus a brand new system to handle 1.8.9's blockstates.

Things are going rather quickly, so I think I actually might be able to finish the update to 1.8.9 in a week at most, assuming I don't run into another major hurdle.

Sorry if I kept you waiting.
